### PR TITLE
Update reload to listen for annotation data changes

### PR DESCRIPTION
**Pull request by martocodo**

### CODE REVIEW

File: src/utils/mouseOver.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import Color from 'color';
+// import Color from 'color';
```
Comment (RMHonor) [inline src/utils/mouseOver.ts]: Safe to remove?

File: src/utils/mouseOver.ts [hunk 2]
```diff
@@ -36,18 +36,22 @@ export const onMouseOver = (props: IProps) => (d: IGroupDataItem | any, i: numbe
   } = props;
   const ix = bins.findIndex((b) => b === d.label);
   // Initial color
-  let color = Color(colors(d.colorRef || String(i)));
+  // let color = Color(colors(d.colorRef || String(i)));
```
Comment (RMHonor) [inline src/utils/mouseOver.ts]: As above

File: src/utils/mouseOver.ts [hunk 3]
```diff
@@ -36,18 +36,22 @@ export const onMouseOver = (props: IProps) => (d: IGroupDataItem | any, i: numbe
...
   if (hover) {
+    // @TODO need to investigate an alternative way of doing this with changing counts
```
Comment (RMHonor) [inline src/utils/mouseOver.ts]: Looking at this, are we losing colours from histograms?

File: src/utils/mouseOver.ts (hunk 2 above)
Comment (martocodo) [reply src/utils/mouseOver.ts]: Yep can also be removed

File: src/utils/mouseOver.ts (hunk 3 above)
Comment (martocodo) [reply src/utils/mouseOver.ts]: Nope this was just applying a colour variant to the bars on hover and restoring it on mouse exit, but this was causing issues with the colour scheme on the overlaid charts. I've simplified it to instead use a short transition and opacity change to help indicate the active bar

**Pull request merged**